src/components/Dragons/Dragons.js

```diff
@@ -0,0 +1,42 @@
+import { useSelector } from 'react-redux';
+
+const Dragons = () => {
+  const dragons = useSelector((state) => state.dragonsReducer.dragons);
+  return (
+    <div>
+      {dragons.map((dragon) => {
+        const {
+          id, name, description, image, reserved,
```

Comment:
[REQUIRED] Using "reserved" here is not necessary because you did not fetch it from the API; consider removing it or going back to fetch it in apiFunctions.js